frontend/ui/widget/textviewer.lua

```diff
@@ -67,28 +70,38 @@ local TextViewer = InputContainer:extend{
     add_default_buttons = nil,
     default_hold_callback = nil, -- on each default button
     find_centered_lines_count = 5, -- line with find results to be not far from the center
+
+    text_category = "general",
+    text_categories = {
+        general      = { monospace_font = false, font_size = 20, justified = false },
+        file_content = { monospace_font = false, font_size = 20, justified = false },
+        book_info    = { monospace_font = false, font_size = 20, justified =  true },
+        bookmark     = { monospace_font = false, font_size = 20, justified =  true },
+        dictionary   = { monospace_font = false, font_size = 20, justified = false },
```

Comment:
This "dictionary" does not impact any dictionary result I think (we don't use TextViewer for them). It would be used only for translator currently.
So, may be "lookup_result" or "lookup" ?